cspell.json

```diff
@@ -17,7 +17,8 @@
     "docs/native",
     "versioned_docs/**/api",
     "versioned_docs/**/cli",
-    "versioned_docs/**/native"
+    "versioned_docs/**/native",
+    "static/code/stackblitz/**/node_modules/**"
```

Comment:
Should we just include all `node_modules`?

Reply:
Let me see. I know it excludes the root `node_modules` directory by default, but doesn't seem to account for nested folder structures with it.

Reply:
Silly that manually specifying it works 🤷 addressed here: https://github.com/ionic-team/ionic-docs/pull/3165/commits/874299f7e89049c0770417c08def4cbb042cae39